File: {{cookiecutter.project_slug}}/src/main/java/{{ cookiecutter.package_path }}/control/Controls.java
package {{cookiecutter.package_name}}.control;

public class Controls {

{% if cookiecutter.use_driver_controls == 'y' %}
  private final DriverControls driverControls = new DriverControls(0);

  public DriverControls getDriverControls() {
    return driverControls;
  }
{% endif %}

{% if cookiecutter.use_game_controls == 'y' %}
  private final GameControls gameControls = new GameControls(1);

  public GameControls getGameControls() {
    return gameControls;
  }
{% endif %}

}

File: {{cookiecutter.project_slug}}/src/main/kotlin/{{ cookiecutter.package_path }}/control/Controls.kt
package {{cookiecutter.package_name}}.control

class Controls {

{% if cookiecutter.use_driver_controls == 'y' %}
    val driverControls = DriverControls(0)
{% endif %}

{% if cookiecutter.use_game_controls == 'y' %}
    val gameControls = GameControls(1)
{% endif %}

}
